### ISSUE TITLE
[feature]  time-stops

### ISSUE THREAD

**Issue by AkliOnet** [taniman/profit-trailer#366]
ALL_stop_loss_trigger_max_hours 
ALL_stop_loss_trigger_trailling 
ALL_stop_loss_trigger_min_sell_value = -5

to complete the ALL_stop_loss_trigger for a full strategy againt coins that don't sell. The idea is that as the btc market alsways go down after day after day it's more profitable to sell with a little loss to refill the balance and continue the trade but only by limiting the loss.

**Comment by taniman** [comment]
what does ALL_stop_loss_trigger_min_sell_value do?

**Comment by taniman** [comment]
now I see better. You are trying to add 3 new functionality in 1 request. I am not a fan of that

**Comment by AkliOnet** [comment]
I did not well explained this one. ALL_stop_loss_trigger_max_loss = -5

**Comment by AkliOnet** [comment]
No problem Elroy .. I stop here ^^

**Comment by taniman** [comment]
Why do you need a ALL_stop_loss_trigger_trailling if you already have the ALL_stop_loss_trigger_max_los

can max_loss not be combined with trailing profit? somehow? or am I missing something?

**Comment by AkliOnet** [comment]
The thing is that I see that the coin with max double up (4->6),  day after day have more and more difficulty to approch the positivity. 

So Day 1 the bags reach the -3% but don't sell of course, then day 2 it reach max 5 , then i will need to wait one week or more to reach the -5%

So the idea is to try to sell at loss for coin that did not sell after 48h by starting to try to sell at -5 but trail (with 0.15 for example) to try to recover the maximum. If the trailling allow to sell at -3.7 it's better than directly sell at -5%  . And as mentionned the goal is to refill the balance so little bags close to the positivity can dca an to continue to trade, so this only make sense if the bags is big so another property is need ALL_stop_loss_trigger_min_cost = 0.03.

**Comment by AkliOnet** [comment]
But there is a chance that the trailing of this strategy conflict with the dca trailling . To not conflict this ha strategy need to stop at the sell value of the dca setting. But if the range idea is retained all this can be added in the dca, so only 

ALL_stop_loss_trigger_max_hours
ALL_stop_loss_trigger_min_cost 
need to be added .

And we will only set this setting to the last range (range_5 for example), so after all the dca trigger have been done.

**Comment by YoJeff** [comment]
We have a variant of timestops in v2.x,  add any further request details to the linked enhancement request.
